srtm4.py

```diff
@@ -19,7 +19,7 @@
 
 BIN = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'bin')
 GEOID = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'data')
-SRTM_DIR = os.path.join(os.path.expanduser('~'), '.srtm')
+SRTM_DIR = os.getenv('SRTM_DIR')
```

Comment:
I'd suggest keeping the default `~/.srtm` cache directory if the env var is not set
```suggestion
SRTM_DIR = os.getenv('SRTM_DIR')
if SRTM_DIR is None:
    SRTM_DIR = os.path.join(os.path.expanduser('~'), '.srtm')
```